public/components/custom_panels/home.tsx

```diff
@@ -98,7 +100,7 @@ export const Home = ({
   };
 
   const onEditClick = (savedVisualizationId: string) => {
-    window.location.assign(`#/event_analytics/explorer/${savedVisualizationId}`);
+    window.location.assign(`observability-logs#/${savedVisualizationId}`);
```

Comment:
observabilityLogsID instead of observability-logs? and why remove `/explorer`? how does event analytics know if it's in the table of saved queries page or PPL query results page

Reply:
Apart from this : LGTM